.github/workflows/ci.yml

```diff
@@ -630,15 +630,15 @@ jobs:
         hadoop fs -put ${GS_TEST_DIR}/ldbc_sample/person_knows_person_0_0.csv /ldbc_sample/person_knows_person_0_0.csv
 
         cd ${GITHUB_WORKSPACE}/python
-        python3 -m pytest -s -vvv tests/kubernetes/test_store_service.py -k test_demo_unloaded
+        python3 -m pytest -s -vvv tests/kubernetes/test_store_service.py -k test_demo
 
         # helm restart ci release and run demo with the PersistentVolume
         # FIXME(@haixia): PersisitentVolume not work
-        # helm uninstall ci
-        # helm install ci --set image.tag=${{ github.sha }},executor=${{ matrix.executor }} \
-        #                               ./graphscope-store
-        # helm test ci --timeout 5m0s
-        # python3 -m pytest -s vvv tests/kubernetes/test_store_service.py -k test_demo_loaded
+        helm uninstall ci
+        helm install ci --set image.tag=${{ github.sha }},executor=${{ matrix.executor }} \
+                                      ./graphscope-store
+        helm test ci --timeout 5m0s
+        python3 -m pytest -s vvv tests/kubernetes/test_store_service.py -k test_loaded_demo
```

Comment:
The name seems not intuitive... What about `test_demo_after_restart` and `test_demo_from_scratch`